opensrp-chw/src/main/java/org/smartregister/chw/reporting/ChwReport.java

```diff
@@ -61,7 +61,7 @@ public static void showIndicatorVisualisations(ViewGroup mainLayout, List<Map<St
         NumericDisplayModel indicator8 = ReportingUtil.getIndicatorDisplayModel(ReportContract.IndicatorView.CountType.LATEST_COUNT, ReportingConstants.ChildIndicatorKeys.DECEASED_CHILDREN_0_11_MONTHS, R.string.deceased_children_0_11_months, indicatorTallies);
         appendView(mainLayout, new NumericIndicatorView(mainLayout.getContext(), indicator8));
 
-        NumericDisplayModel indicator9 = ReportingUtil.getIndicatorDisplayModel(ReportContract.IndicatorView.CountType.LATEST_COUNT, ReportingConstants.ChildIndicatorKeys.DECEASED_CHILDREN_12_59_MONTHS, R.string.deceased_children_12_59_months, indicatorTallies);
```

Comment:
Will affect all variants. Implements a cut out date on the global flavor interface